cypress/integration/26-register_registration/creator_begins_registering_a_registration/begin_registering_registration.js

```diff
@@ -33,11 +33,7 @@ And('they have selected {string} for starting the Wizard', (method) => {
 });

```

Comment:
^ btw: til `And` over her har det kommet enda en måte å starte registrering nå. Scenario er vel i PR atm

Reply:
Tar tak i det når testene feiler...